frontend/models.py

```diff
@@ -18,8 +18,10 @@
 """
 
 
+from tkinter import CASCADE
```

Comment:
This line is not needed (you were looking for models.CASCADE)

Reply:
Done!